features/attached_status.feature

```diff
@@ -7,18 +7,13 @@ Feature: Attached status
         Given a `<release>` machine with ubuntu-advantage-tools installed
         When I attach `contract_token` with sudo
         And I run `ua status --format json` as non-root
-        Then stdout is formatted as `json` and has keys:
-            """
-            _doc _schema_version account attached config config_path contract effective
-            environment_vars execution_details execution_status expires machine_id notices
-            services version simulated
-            """
+        Then stdout is a json matching the `ua_status` schema
         When I run `ua status --format yaml` as non-root
         Then stdout is formatted as `yaml` and has keys:
             """
             _doc _schema_version account attached config config_path contract effective
             environment_vars execution_details execution_status expires machine_id notices
-            services version simulated
+            services version simulated result errors warnings
             """
```

Comment:
Not necessary for this PR, but we could probably write a step function that reads like:
```
Then stdout is yaml matching the `ua_status` schema
```

Reply:
Can be done in this PR, good call. We can then get rid of this `is formatted as` verification.

Reply:
Created SC-797 for this one

Reply:
@orndorffgrant I actually found a feasible way to do this! So we don't need another PR - it is in a separate commit to facilitate reviewing/changing/removing as needed.